backend/dal/Migrations/20210128205552_v01.06.07.cs

```diff
@@ -0,0 +1,22 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+using Pims.Dal.Helpers.Migrations;
+
+namespace Pims.Dal.Migrations
+{
+    public partial class v010607 : SeedMigration
```

Comment:
Sorry @jdtoombs need to update to v01.07.00 now for Sprint 23

Reply:
woops just saw this @Fosol will do!